Cargo.toml

```diff
@@ -27,6 +27,10 @@ members = [
 ]
 exclude = ["test_util/std/hash/_wasm"]
 
+# use serde_json lucacasonato/json.git@5ec8141acf127d9dcbc01b2b63c6426d158ee1af
+[patch.crates-io]
+serde_json = { git = "https://github.com/lucacasonato/json.git", rev = "5ec8141acf127d9dcbc01b2b63c6426d158ee1af" }
+
```

Comment:
Will be removed when https://github.com/serde-rs/json/pull/830 is landed and released.